api/user/passport-initializer.ts

```diff
@@ -22,8 +27,13 @@ export default class PassportInitializer {
   install(app: Express): void {
     // Add middleware that sends and receives the session ID using cookies
     // See https://github.com/expressjs/session#readme
+    const RedisStore = connectRedis(session)
     app.use(
       session({
+        store: new RedisStore({
+          client: this.redisClient,
+          disableTouch: true,
```

Comment:
Why did you disabled touch, i.e that session are kept alive when the user interacts with the service without changing the session in itself?

Reply:
In Redis, we can store data with a timer, and then later, whenever we interact with that data, Touch will update the timer. Here we aren't setting any timer. We want to store sessions forever. So no need to keep updating it.
It reduces the number of requests to Redis.

Reply:
Okay, I understood the documentation
> ttl
If the session cookie has a expires date, connect-redis will use it as the TTL.
Otherwise, it will expire the session using the ttl option (default: 86400 seconds or one day).

that it normally expires after 1day. If `disableTouch: false` then actions reset the ttl, so that session expire 1 day after the last interaction; otherwise 1 day after creation of the session. Anyway, since maxAge is set rather high, this shouldn't be much of a problem and can easily be changed later.